### ISSUE TITLE
Deploy Latest Release to Instruments (Group A)

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#7538]
As someone who uses and supports IBEX I want the systems kept up to date so having built a release we should deploy it. (In theory this should be Group A, see https://github.com/ISISComputingGroup/IBEX/wiki/Release-Group, but the list will be updated during planning, adding and removing as appropriate.)

## Acceptance Criteria
- [x] The instruments below are reformatted to allow checking for "in progress" and "done"
- [ ] The instruments listed below are either "done", or where delayed a comment and a new ticket are created as appropriate

## Instruments to Deploy to
IMAT
IMAT-UCBDETECT (32 bit, double check)
ENGINX
ENGINX_SETUP
ENGINX-STRESS1 (32 bit, double check)
CRISP
INTER
OFFSPEC
POLREF
SURF
LARMOR
LOQ
SANS2D
ZOOM
GEM
HRPD
HRPD_SETUP
INES
PEARL
PEARL_SETUP
POLARIS
WISH
WISH_SETUP
EMMA
DETMON
LET
NIMROD
SANDALS
RIKENFE
VESUVIO
SOFTMAT

**Comment by rerpha** [comment]
LARMOR and SANS2D will need to have motorExtensions.cmd files copied over from https://github.com/ISISComputingGroup/IBEX/issues/7539

**Comment by FreddieAkeroyd** [comment]
All TS2 is on an older ibex version, so we should do all TS2 + whatever we can manage in TS1. Science Groups often like to have all the instruments in the group on the same version of ibex, this will likely dictate some of the TS1 subset.

**Comment by FreddieAkeroyd** [comment]
* May be useful to do VESUVIO rather than hotfix maclennan updates
* crisp/surf to keep in line with ts2 reflectometers and also for jaws update (but jaws is just db so can be easily hotfixed)
* sandals for new beckhoff

**Comment by FreddieAkeroyd** [comment]
@KathrynBaker OK to add VESUVIO for them to benefit from https://github.com/ISISComputingGroup/IBEX/issues/7471 and
https://github.com/ISISComputingGroup/IBEX/issues/7477 ?

**Comment by KathrynBaker** [comment]
@FreddieAkeroyd Go ahead

**Comment by rerpha** [comment]
IMAT
- [ ] In progress
- [ ] Done
IMAT-UCBDETECT (32 bit, double check)
- [ ] In progress
- [ ] Done
ENGINX
- [ ] In progress
- [ ] Done
ENGINX_SETUP
- [ ] In progress
- [ ] Done
ENGINX-STRESS1 (32 bit, double check)
- [ ] In progress
- [ ] Done
CRISP
- [ ] In progress
- [ ] Done
INTER
- [ ] In progress
- [ ] Done
OFFSPEC
- [ ] In progress
- [ ] Done
POLREF
- [ ] In progress
- [ ] Done
SURF
- [ ] In progress
- [ ] Done
LARMOR
- [ ] In progress
- [ ] Done
LOQ
- [ ] In progress
- [ ] Done
SANS2D
- [ ] In progress
- [ ] Done
ZOOM
- [ ] In progress
- [ ] Done
GEM
- [ ] In progress
- [ ] Done
HRPD
- [ ] In progress
- [ ] Done
HRPD_SETUP
- [ ] In progress
- [ ] Done
INES
- [ ] In progress
- [ ] Done
PEARL
- [ ] In progress
- [ ] Done
PEARL_SETUP
- [ ] In progress
- [ ] Done
POLARIS
- [ ] In progress
- [ ] Done
WISH
- [ ] In progress
- [ ] Done
WISH_SETUP
- [ ] In progress
- [ ] Done
EMMA
- [ ] In progress
- [ ] Done
DETMON
- [ ] In progress
- [ ] Done
LET
- [ ] In progress
- [ ] Done
NIMROD
- [ ] In progress
- [ ] Done
SANDALS
- [ ] In progress
- [ ] Done
RIKENFE
- [ ] In progress
- [ ] Done
VESUVIO
- [ ] In progress
- [ ] Done

**Comment by NikolaRoev** [comment]
Added SOFTMAT to deploy schedule for T-Jump test. Waiting on reply for a time to do the deploy.

**Comment by rerpha** [comment]
task for IMAT-UCBDETECT: https://github.com/ISISComputingGroup/IBEX/projects/8#card-87800543
When DETMON is done I will stick this in review
